apps/server/src/modules/provisioning/strategy/sanis/sanis.strategy.ts

```diff
@@ -69,6 +81,18 @@ export class SanisProvisioningStrategy extends OidcProvisioningStrategy {
 		return oauthData;
 	}
 
+	private async checkResponseValidation(response: SanisResponse, groups: SanisResponseValidationGroups[]) {
+		const validationErrors: ValidationError[] = await validate(response, {
+			always: true,
+			forbidUnknownValues: false,
+			groups,
+		});
+
+		if (validationErrors.length) {
+			throw new ValidationErrorLoggableException(validationErrors);
```

Comment:
we talked with bernhard yesterday and discussed how to handle all constellations of missing data and we agreed that when the validation fails because the response is not processable on our side then we throw an error and block the login. https://docs.dbildungscloud.de/x/8QHIEQ